src/Cell.ts

```diff
@@ -54,18 +55,23 @@ export enum CellType {
   VALUE = 'VALUE',
   ARRAY = 'ARRAY',
   EMPTY = 'EMPTY',
+  ARRAYFORMULA = 'ARRAYFORMULA',
```

Comment:
Pls add it on docs page as well: https://github.com/handsontable/hyperformula/blob/master/docs/guide/types-of-values.md